### PR TITLE
added working redux boilerplate

### PR DESCRIPTION
**Pull request by brian-z-liang**
Added redux boilerplate

### CODE REVIEW

File: client/index.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
+import { Provider } from 'react-redux';
+import App from './components/App.jsx';
+import store from './store';
 
 render(
-  <div>
-    <h1>It worked.</h1>
-  </div>,
+  // wrap the App in the Provider Component and pass in the store
+  <Provider store={store}>
```
Comment (sturgeonphillip) [inline client/index.js:9]: App nested between Provider.

**Review by ajsmith925 — APPROVED** [review]
Looks solid!

File: client/index.js (hunk 1 above)
Comment (sturgeonphillip) [reply client/index.js:9]: approved.

**Review by sturgeonphillip — APPROVED** [review]
approved.

**Pull request merged**